package.json

```diff
@@ -67,6 +69,8 @@
     "wdio": "cross-env NODE_OPTIONS=--max-old-space-size=2048 wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
   },
   "devDependencies": {
+    "@babel/register": "^7.0.0",
+    "axe-puppeteer": "^0.1.0",
```

Comment:
This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.